riscv64-purecap.cmake

```diff
@@ -0,0 +1,23 @@
+set(CMAKE_SYSTEM_NAME Linux)
+
+#Set toolchain compilers
```

Comment:
I tried to suppress my OCD, but I have failed :) Can we consistently use a space character after the `#`? :) Also, we're sometimes using one, and sometimes two, newlines between sections, and I wasn't sure if that implied bigger "semantic" differences or not -- I *think* not, but I might be wrong!